examples/containers/docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
         container_name: couchbase-cmos
         # Note an environment file can be used as well, this is likely better for production
         environment:
+            - DISABLE_PROMTAIL=true # See CMOS-179
```

Comment:
Not necessary since https://github.com/couchbaselabs/observability/pull/120 was merged